HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -127,33 +127,55 @@ private void saveMap()
 
         private void Edit_Display_Information(object sender, RoutedEventArgs e)
         {
+            this.chooser.SelectedItem = DisplayMapObject.MapObjectType.ObjectTypeFullName;
             EditMode = !EditMode;
         }
 
         public void ChangeEditButtonVisibility() {
             if (_selectedMapObject == null)
             {
                 EditObjectButton.Visibility = Visibility.Hidden;
+                Save.Visibility = Visibility.Hidden;
+                EditMode = false ;
             }
             else
             {
                 if (_selectedMapObject.MapObjectEntity.MapObjectType.TypeOfMapObject != TypeOfMapObject.ROAD)
                 {
                     EditObjectButton.Visibility = Visibility.Visible;
+                    Save.Visibility = Visibility.Visible;
                 }
                 else {
                     EditObjectButton.Visibility = Visibility.Hidden;
+                    Save.Visibility = Visibility.Hidden;
                 }
             }
         }
 
+        private MapObjectType SettingTypeOfEditedMapObject() {
+            MapObjectType type;
+            if (chooser.SelectedItem != null)
```

Comment:
@kmilena98 Also complete "chooser" with a little bit more information? Is it a Combo Box? What you choose with it?
Maybe instead of "chooser" name it "ObjectTypeChooserComboBox" or something like that...

Reply:
If this is resolved, 'resolve conversation' @kmilena98 @VasilijeBursac 

![image](https://user-images.githubusercontent.com/45834270/99149024-c3125400-268b-11eb-8bef-2eb7d23b9f91.png)